packages/osd-monaco/src/monaco_environment.ts

```diff
@@ -7,11 +7,13 @@ import { getWorker } from './worker_store';
 
 // @ts-ignore
 window.MonacoEnvironment = {
-  getWorker: (module: string, languageId: string) => {
-    const workerSrc = getWorker(languageId);
+  getWorker: (workerId: string, label: string) => {
```

Comment:
nit: for developers i can safely assume that if i pass an id to a function `getWorker` it would be assumed that is the worker ID so it's a little bit redundant.

however from what i can tell that the workerId is actually not used so to be i think i preferred the discarded parameter convention of using an underscore `_` to indicate that we are not using the id. this will indicate to developers that we actually don't care for the worker Id

Reply:
good point. will change.